cookies.py

```diff
@@ -0,0 +1,45 @@
+import pickle
+import pprint
+import time
+
+from selenium import webdriver
+
+## Using selenium to bypass JSTOR Login 
+
+
+def save_cookies(driver, location):
+
```

Comment:
can you remove this space. The convention is not to have spaces after function definition